api.go

```diff
@@ -135,6 +136,22 @@ func (m *FileMatch) sizeBytes() (sz uint64) {
 	return
 }
 
+// addScore increments the score of the FileMatch by the computed score. If
+// debugScore is true, it also adds a debug string to the FileMatch. If raw is
+// -1, it is ignored. Otherwise, it is added to the debug string.
+func (m *FileMatch) addScore(what string, computed float64, raw float64, debugScore bool) {
```

Comment:
I moved this here from `score.go` so all `FileMatch` methods are in the same place.